Classes/Controller/DocumentFormController.php

```diff
@@ -47,39 +61,30 @@ public function newAction(\EWW\Dpf\Domain\Model\DocumentForm $newDocumentForm =
      */
     public function createAction(\EWW\Dpf\Domain\Model\DocumentForm $newDocumentForm)
     {
-        foreach ($newDocumentForm->getFiles() as $file) {
-            $uid = $file->getUID();
-            if (empty($uid)) {
-                $file->setDownload(true);
-            }
-            $files[] = $file;
-        }
-
-        $newDocumentForm->setFiles($files);
+        // Replaced by the form configuration (download/archive)
```

Comment:
Commented out code. Can it be removed?

Reply:
The outcommented code has been removed. A download checkbox field must be configured for the input assistant (front-end form), otherwise the files will not be listed as downloadable.